packages/core/src/auth/credential-provider.ts

```diff
@@ -69,6 +69,16 @@ export abstract class CredentialProvider {
    */
   abstract isCacheEndpointSecure(): boolean;
 
+  /**
+   * @returns {string} The host which the Momento client will connect to for Momento storage operations
+   */
+  abstract getStorageEndpoint(): string;
+
+  /**
+   * @returns {boolean} true if connecting to the storage endpoint connection without TLS; false if using TLS
```

Comment:
```suggestion
   * @returns {boolean} true if connecting to the storage endpoint connection with TLS; false if not using TLS
```
Oops, I think I had mixed around the wording when I added this. Probably need to fix wherever this was copied from as well